modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -6,25 +6,100 @@ label:AuraDB-Virtual-Dedicated-Cloud[]
+
+== Setup requirements
+
+Accessing Aura with SSO requires:
+
+* Authorization Code Flow
+* A publicly accessible IdP server
+
+To create an SSO Configuration either a Discovery URI or a combination of Issuer, Authorization Endpoint, Token Endpoint and JWKS URI is required.
+
+== Create a new SSO configuration
+
+From the *Organization settings*, go to *Single Sign-On* and use the *SSO Configuration* button to set up a new SSO configuration.
+
+.Organization settings
+[.shadow]
+image::organizationsettings.png[A screenshot of how to navigate to organization settings in the UI]
+
+The checkboxes *Use as a log in for the Organization* and *Use as login method for instances with projects in this Org* define whether SSO should be only on Organization level, only on Project level, or both.
+
+The required basic SSO configuration information can be retrieved from the IdP.
+Entering the Discovery URI pre-fills the fields below. 
+If this is not known these fields can be completed manually.
+
+.SSO configuration
+[.shadow]
+image::sso.png[A screenshot of the SSO configuration dialogue,640,480]
+
+.SSO toggles
+[.shadow]
+image::ssotoggles.png[A screenshot of the SSO toggles (for password/user and Google social log in) after SSO configuration dialogue,640,480]
+
+== Individual instance level SSO configurations available from Support
+
+Support can assist with:
+
+* Role mapping specific to a database instance
+* Custom groups claim besides `groups`
+* Updating SSO on already running instances
+
+If you require support assistance, visit link:https://support.neo4j.com/[Customer Support] and raise a support ticket including the following information:
+
+
+. The _Project ID_ of the projects you want to use SSO for.
+See xref:platform/user-management.adoc#_projects[Projects] for more information on how to find your __Project ID__.
+
+. The name of your IdP
+
+== Authorization vs. Authentication
+
+*Authentication* determines who is allowed in. 
+Authentication is handled by Single Sign-On (SSO), which serves as a method for logging in. 
+
+*Authorization* determines what someone can do when they have logged-in.
+Access privileges beyond login are managed through roles using Role-Based Access Control (RBAC) an authorization method. 
 
+SSO does not directly grant access to edit project settings—such as changing the project name, managing network access, or modifying instance settings like renaming, pausing, or resuming an instance. 
+To determine a user’s access rights to these features, you should use RBAC.
 
-=== SSO Org level roles
+Roles and permissions are managed by RBAC, which decides whether a user can access, view, or modify data within the database instances themselves. 
+At this level, role mapping can be utilized to grant users different levels of access based on their roles in their Identity Provider (IdP).
 
-The following roles are available at the org level and these are assigned via invitation:
+=== RBAC determined roles related to SSO
```

Comment:
What does this mean?

Reply:
Are these roles assigned through SSO configurations and invitations, or are they are assigned separately through RBAC configurations and invitations?

Could it be more like **You can map your IdP groups to Aura roles. The following roles are available:**

It would be cool if we could try it out and see how it works!